Module: transfer_money.py 
Function: process_transfer 
Error: ProgrammingError(1146, "Table 'bankapp_db.intotransaction' doesn't exist")